---
 configure |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: eglibc-2.10/configure
===================================================================
--- eglibc-2.10.orig/configure	2009-12-01 13:40:26.000000000 +0100
+++ eglibc-2.10/configure	2009-12-01 13:40:26.000000000 +0100
@@ -4439,7 +4439,7 @@
   echo "$as_me:$LINENO: result: $ac_prog_version" >&5
 echo "${ECHO_T}$ac_prog_version" >&6
 fi
-if test $ac_verc_fail = yes; then
+if test $ac_verc_fail = yes -a -z "$enable_hacker_mode"; then
   critic_missing="$critic_missing gcc"
 fi
 
