---
 libiberty/Makefile.in |    1 +
 1 file changed, 1 insertion(+)

Index: gcc-4.4.1/libiberty/Makefile.in
===================================================================
--- gcc-4.4.1.orig/libiberty/Makefile.in	2009-11-19 13:43:48.000000000 +0100
+++ gcc-4.4.1/libiberty/Makefile.in	2009-11-19 13:45:37.000000000 +0100
@@ -227,6 +227,7 @@
 	  $(AR) $(AR_FLAGS) $(TARGETLIB) \
 	    $(REQUIRED_OFILES) $(EXTRA_OFILES) $(LIBOBJS); \
 	  $(RANLIB) $(TARGETLIB); \
+	  cp $(TARGETLIB) ../ ; \
 	  cd ..; \
 	else true; fi
 
